src/shape/custom_shapes.js

```diff
@@ -1625,24 +2052,63 @@ function customShapes(p5, fn) {
   };
 
   /**
-   * @method splineProperty
```

Comment:
Looks like we lost this overload block, can we add this one back after the current one?